docs/root/intro/arch_overview/load_balancing.rst

```diff
@@ -247,6 +227,72 @@ To sum this up in pseudo algorithms:
+assumed that default value of 50% is used for panic threshold.
+
+Assume a simple set-up with 2 priority levels, P=1 100% healthy. In this scenario
+total health is always 100% and P=0 never enters panic mode and Envoy is able to shift entire traffic to P=1.
+
++----------------------------+--------------------+
+| P=0 healthy endpoints      | P=0 in panic       |
++============================+====================+
+| 100%                       | NO                 |
++----------------------------+--------------------+
+| 72%                        | NO                 |
++----------------------------+--------------------+
+| 71%                        | NO                 |
++----------------------------+--------------------+
+| 50%                        | NO                 |
++----------------------------+--------------------+
+| 25%                        | NO                 |
++----------------------------+--------------------+
+| 0%                         | NO                 |
++----------------------------+--------------------+
+
+If P=1 becomes unhealthy, panic threshold continues to be disregarded until the sum of the health
+P=0 + P=1 goes below 100. At this point Envoy starts checking panic threshold value for each 
+priority.
+
++------------------------+-------------------------+-----------------+-----------------+-----------------+
+| P=0 healthy endpoints  | P=1 healthy endpoints   | Total health    |  P=0 in panic   | P=1 in panic    |
++========================+=========================+=================+=================+=================+
+| 100%                   |  100%                   | 100%            |   NO            | NO              | 
++------------------------+-------------------------+-----------------+-----------------+-----------------+
+| 72%                    |  72%                    | 100%            |   NO            | NO              |
++------------------------+-------------------------+-----------------+-----------------+-----------------+
+| 71%                    |  71%                    | 100%            |   NO            | NO              |
++------------------------+-------------------------+-----------------+-----------------+-----------------+
+| 50%                    |  50%                    | 100%            |   NO            | NO              |
++------------------------+-------------------------+-----------------+-----------------+-----------------+
+| 25%                    |  100%                   | 100%            |   NO            | NO              |
++------------------------+-------------------------+-----------------+-----------------+-----------------+
+| 25%                    |  25%                    | 70%             |   YES           | YES             |
++------------------------+-------------------------+-----------------+-----------------+-----------------+
+| 5%                     |  65%                    | 98%             |   YES           | NO              |
```

Comment:
I wasn't in the original discussion, but can you TL;DR summarize what the thinking was in allowing P=0 to enter panic (and hence spray to unhealthy host) while P=1 seems happier to sink the traffic in this case?

Reply:
(and maybe try and convey this intuition regarding it being the sensible thing to do in the writeup here)

Reply:
Tried my best.